storybook/src/components/FileInput/FileInput.stories.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { FileInput } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Components/Forms/File Input',
+  component: FileInput,
+  args: {
+    accept: '',
```

Comment:
Don't we want this to be `undefined` initially? It renders an empty `accept` attribute now.